### PR TITLE
Compatibility - Add Hatchet Compat

### PR DESCRIPTION
**Pull request by SpicyBagpipes**
**When merged this pull request will:**
- title

### Important
- [x] If the contribution affects [the documentation](https://github.com/DartsArmaMods/AimForTheBushes/tree/main/docs), please include your changes in this pull request.
- [x] [Development Guidelines](https://github.com/DartsArmaMods/AimForTheBushes/tree/main/.github/CONTRIBUTING.md) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### Known Issues
N/A

### CODE REVIEW

**Comment by SpicyBagpipes** [comment]
still wip

**Comment by SpicyBagpipes** [comment]
Done and tested @DartRuffian

File: .vscode/sqfvm-lsp/ls-ignore.txt
Comment (DartRuffian) [inline .vscode/sqfvm-lsp/ls-ignore.txt:1]: Gonna guess this is an accident

File: addons/compat_hatchet/config.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        author = AUTHOR;
+        authors[] = {"DartRuffian","KJW"};
```
Comment (DartRuffian) [inline addons/compat_hatchet/config.cpp:6]:
```suggestion
        authors[] = {"KJW"};
```

**Comment by DartRuffian** [comment]
Don't forget to update `docs/README.md` to add the mod name

File: .vscode/sqfvm-lsp/ls-ignore.txt
Comment (SpicyBagpipes) [reply .vscode/sqfvm-lsp/ls-ignore.txt:1]: oops thought you had .vscode in gitignore already

**Comment by SpicyBagpipes** [comment]
> Don't forget to update `docs/README.md` to add the mod name and link (the one in the repository root is updated on release)

fixed

File: .vscode/sqfvm-lsp/ls-ignore.txt
Comment (DartRuffian) [reply .vscode/sqfvm-lsp/ls-ignore.txt:1]: Probably should, since vs code settings can just be grabbed using the workspace

**Review by DartRuffian — APPROVED** [review]

**Pull request merged**